llvm/lib/Target/X86/Disassembler/X86DisassemblerDecoder.h

```diff
@@ -33,13 +33,24 @@ namespace X86Disassembler {
 #define xFromREX(rex)        (((rex) & 0x2) >> 1)
 #define bFromREX(rex)        ((rex) & 0x1)
 
+#define mFromREX2(rex2) (((rex2) >> 7) & 0x1)
+#define r2FromREX2(rex2) (((rex2) >> 6) & 0x1)
+#define x2FromREX2(rex2) (((rex2) >> 5) & 0x1)
+#define b2FromREX2(rex2) (((rex2) >> 4) & 0x1)
+#define wFromREX2(rex2) (((rex2) >> 3) & 0x1)
+#define rFromREX2(rex2) (((rex2) >> 2) & 0x1)
+#define xFromREX2(rex2) (((rex2) >> 1) & 0x1)
+#define bFromREX2(rex2) ((rex2) & 0x1)
```

Comment:
Align with the code around, and use & 0xXX >> X format to align.

Reply:
Not addressed?

Reply:
Addressed just now. I submitted a NFC patch and now it's aligned.